components/SingleReport.vue

```diff
@@ -54,9 +65,7 @@ const createLabelsFromCriteria = (currentCriterion) => {
               :name="props.report.name"
               :labels="createLabelsFromCriteria(currentCriteria)"
             />
-            <p class="whitespace-pre-wrap">
-              {{ criteria[currentCriteria] }}
-            </p>
+            <div v-html="getMarkdownContent(criteria[currentCriteria])" />
```

Comment:
**question:** Just to double check — we are sure `marked` is handling the escaping & sanitization?

Reply:
AAA

no

<img width="898" alt="CleanShot 2022-12-07 at 11 19 48@2x" src="https://user-images.githubusercontent.com/1051509/206247086-5d9c7b09-ab79-4a06-8561-737886de28c2.png">


Will add

Reply:
Done!